packages/launch/src/focused-launch/index.tsx

```diff
@@ -2,26 +2,32 @@
  * External dependencies
  */
 import React from 'react';
-import { MemoryRouter as Router, Switch, Route } from 'react-router-dom';
+import { MemoryRouter as Router, Switch, Route, Redirect } from 'react-router-dom';
 
 /**
  * Internal dependencies
  */
+import { useSite } from '../hooks';
 import { Route as FocusedLaunchRoute } from './route';
 import Summary from './summary';
 import DomainDetails from './domain-details';
 import PlanDetails from './plan-details';
 import Success from './success';
-import { useOnLaunch } from '../hooks';
 
 import './style.scss';
 
 const FocusedLaunch: React.FunctionComponent = () => {
-	// handle redirects to checkout / my home after launch
-	useOnLaunch();
+	const { isLaunched, isSiteLaunching } = useSite();
+
+	React.useEffect( () => {
+		if ( isLaunched || isSiteLaunching ) {
+			document.body.classList.add( 'is-focused-launch-complete' );
```

Comment:
Maybe a better alternative is not to append the launch button at all in https://github.com/Automattic/wp-calypso/blob/a669343f83369e6062273e009416d874c70a3aa1/apps%2Fediting-toolkit%2Fediting-toolkit-plugin%2Feditor-gutenboarding-launch%2Findex.ts#L156

Reply:
The button is already appended at this time and we don't refresh the page is the user is choosing to "continue editing"

Reply:
I see. I was hoping we could have a less hacky way of hiding the Launch button than adding a class to the `body`

Reply:
Yup, I can agree it's not ideal but at least it's cleaner than how we add the button. We'll have the non-hacky solution once we use some Gutenberg API to insert the button properly 😬